storage/innobase/srv/srv0srv.cc

```diff
@@ -532,7 +532,7 @@ ulong srv_n_purge_threads = 4;
 /* the number of pages to purge in one batch */
 ulong srv_purge_batch_size = 20;
 
-ulong srv_encrypt_tables = 0;
+enum_default_table_encryption srv_default_table_encryption;
```

Comment:
Can InnoDB use the server variable instead?

Reply:
I put this as a separate copy as it is frequently checked by encryption threads.
Also because anyways I need to call fil_crypt_set_thread_cnt each time default_table_encryption is set. - I am re- using this to set this innodb copy of server variable.

Reply:
How do you syncronize two copies?

Reply:
It can be only updated from the server level. The server's update function is:
Sys_var_enum_default_table_encryption::global_update . There I can handler function fix_default_table_encryption that will update the innodb's copy of this variable. Also this handler is called in mysqld.cc, right after fix_tablespace_empty_uuid.